src/rwe/RenderService.cpp

```diff
@@ -408,7 +408,7 @@ namespace rwe
                     auto timeSinceSpawn = currentTime - projectile.createdAt;
                     auto fullLifetime = projectile.dieOnFrame.value() - projectile.createdAt;
                     auto percentComplete = static_cast<float>(timeSinceSpawn.value) / static_cast<float>(fullLifetime.value);
-                    auto frameIndex = static_cast<unsigned int>(percentComplete * spriteSeries->sprites.size());
+                    int frameIndex = static_cast<int>(percentComplete * spriteSeries->sprites.size());
```

Comment:
nit: in general I prefer `auto`, but particularly in locations like this because it reduces duplication - it's clear from the right hand side that this will be an `int`.